### PR TITLE
AE2 integration, brought to 1.19.3

### PR DESCRIPTION
**Pull request by NerjalNosk**
(All #29 , but in only one commit, and adapted to 1.19.3)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,6 +45,13 @@ dependencies {
 
 	modCompileOnly "carpet:fabric-carpet:${project.minecraft_version}-${project.carpet_version}"
 
+	// AE2 integration
+	modImplementation ("appeng:appliedenergistics2-fabric:${project.ae2_version}") {
```
Comment (TheDeathlyCow) [inline build.gradle:49]: Should this be `modCompileOnly` again?

File: src/main/resources/data/geodes/worldgen/configured_feature/certus_geode.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+					"Name": "minecraft:water"
+				}
+			},
+			"inner_layer_provider": {
+				"type": "minecraft:simple_state_provider",
+				"state": {
+					"Name": "ae2:quartz_block"
+				}
+			},
+			"alternate_inner_layer_provider": {
```
Comment (TheDeathlyCow) [inline src/main/resources/data/geodes/worldgen/configured_feature/certus_geode.json:17]: The middle layer and alternate inner layer providers do not match the 1.19-1.19.2 version, is this intended, or should those also be updated?

File: build.gradle (hunk 1 above)
Comment (NerjalNosk) [reply build.gradle:49]: Mybad, again x)

File: src/main/resources/data/geodes/worldgen/configured_feature/certus_geode.json (hunk 2 above)
Comment (NerjalNosk) [reply src/main/resources/data/geodes/worldgen/configured_feature/certus_geode.json:17]: Somehow I realized having it the same way in the 1.19-1.19.2 version made too many budding blocks generate, thus the difference. Also I may have been a little lazy around the corners. But tbh, I don't think that's really a problem considering how easy it is to get budding blocks out of a few certus quartz with AE2

**Review by TheDeathlyCow — APPROVED** [review]

**Pull request merged**